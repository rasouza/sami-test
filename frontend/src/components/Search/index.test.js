import React from "react";
import { shallow } from "enzyme";
import Input from './'

jest.mock("react-redux", () => ({
  useSelector: jest.fn(),
  useDispatch: jest.fn()
}));

const loading = false

it("render with default styles", () => {
  const actual = shallow(<Input loading={loading} />);
  expect(actual).toMatchSnapshot();
});