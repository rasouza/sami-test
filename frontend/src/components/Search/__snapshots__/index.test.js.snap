// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render with default styles 1`] = `ShallowWrapper {}`;
